verification/curator-service/ui/README.md

```diff
@@ -34,17 +50,34 @@ US).
 - `admin3`: Optional. Meaning differs by country (e.g., this is municipality in the
 US).
 [See full definition here](https://en.wikipedia.org/wiki/List_of_administrative_divisions_by_country).
-- `gender`: Optional. One of `Female`, `Male`, `Non-binary/Third gender`, or `Other`.
-- `ageRangeStart`: Optional. Number (years) from 0 to 120.
-- `ageRangeEnd`: Optional. Number (years) from 0 to 120.
+
+##### Events
 - `dateConfirmed`: **Required**. Date (MM/DD/YYYY) on which the case was
 confirmed.
+- `confirmationMethod`: Optional. Mechanism by which the case was confirmed
+(e.g., `PCR test`).
 - `hospitalized`: Optional. One of `true` or `false`.
 - `dateHospitalized`: Optional. Date (MM/DD/YYYY) on which the patient was
 hospitalized.
+- `icuAdmission`: Optional. One of `true` or `false`.
+- `dateIcuAdmission`: Optional. Date (MM/DD/YYYY) on which the patient was
+admitted to the ICU.
 - `outcome`: Optional. One of either `Death` or `Recovered`.
 - `dateOutcome`: Optional. Date (MM/DD/YYYY) on which the above outcome
 transpired.
+- `dateSymptomOnset`: Optional. Date (MM/DD/YYYY) on which symptoms first
+presented in the case.
+
+##### Symptoms and preexisting conditions
+- `symptomStatus`: Optional. One of `Symptomatic`, `Asymptomatic`, or
+`Presymptomatic`.
+- `symptoms`: Optional. Delimit multiple symptoms with a semicolon (e.g.,
```

Comment:
Could you note here that the symptoms are only allowed if the symptomStatus is "Symptomatic" or "Presymptomatic". Do we do that check on the API? We check for it only in the UI of the single case entry

Reply:
Stephen suggested in #646 that we'd automatically set `Symptomatic` based on the presence of symptom onset date. I think I'm also setting it if `symptoms` is populated.

It's a good question as to whether we'd enforce these types of relationships in the API. We aren't right now AFAICT, but it might be reasonable for `symptom.status` to be a required field within the `symptoms` stanza.

I don't feel strongly. I was happy to add some convenience autofills to the bulk upload, but we can instead error in some of these situations. Perhaps this is something to talk over (for all upload types) as we tackle the verification CUJ?

Reply:
When talking with Shru through the single case form we decided against all autofill values. Since the data is of upmost importance, we'd rather add a little inconvenience in the UI to ensure the data is submitted as intended. I think we should remove all autofilled values here too.

Reply:
SG, I've left a comment to get Stephen's take.

Reply:
I'd prefer to remove all the autofill added in this PR, since I think it's highly likely we won't want to add it. But if you promise to come back to it I suppose it's okay ;)

Reply:
No trouble at all. We can toss it for now, and update later as desired. I'll change these instances to align with existing checks ("silently don't include nonsensical data but we'll make it error later").